contrib/app_acct_tools/process_records.php

```diff
@@ -148,12 +148,12 @@
 			$data[/* "sess_duration" */] = $record["Acct-Session-Time"]." seconds";
 		} else {
 			/* No the information is missing, let's compute the approx value with the START record timestamp */
-			$res = pg_query_params($dbconn, 'SELECT t_start."recorded_on" as begining, t_end."recorded_on" - t_start."recorded_on" as duration'.
+			$res = pg_query_params($dbconn, 'SELECT t_start."recorded_on" as beginning, t_end."recorded_on" - t_start."recorded_on" as duration'.
 							' FROM (SELECT "recorded_on" FROM "'.$INCOMING.'"  WHERE "Session-Id" = $1 AND "Acct-Session-Id" = $2 AND "Accounting-Record-Type" = 4 ORDER BY "recorded_on" LIMIT 1) as t_end, '.
 							'      (SELECT "recorded_on" FROM "'.$INCOMING.'"  WHERE "Session-Id" = $1 AND "Acct-Session-Id" = $2 AND "Accounting-Record-Type" = 2 ORDER BY "Accounting-Record-Number", "recorded_on" LIMIT 1) as t_start',
 							array($sid, $asid)) or die('Query failed: ' . pg_last_error() . "\n");
 			$vals = pg_fetch_array($result, null, PGSQL_ASSOC) or die('Internal error, unable to compute session time');
-			$data[/* "sess_start" */] = $vals["begining"];
```

Comment:
@sdecugis I meant to add a comment here, but probably forgot to hit the button to submit. Could this be a breaking change?

Reply:
I understand "begining" is not part of permanent records in the database, instead it is a variable used by  pg_query_params().